lymproblems/find_ip.py

```diff
@@ -0,0 +1,31 @@
+"""
+Problem: Find out the IP of dgplug.org and put into /tmp/ip_dgplug.txt file.
+"""
+import os
+import sys
+import socket
+from .utils import system, success, find_path_data
+
+def setup():
+    "Setup find_ip"
+    pass  # Nothing to do.
+
+def verify():
+    "Verify find_ip"
+    try:
+        site_url="www.dgplug.org"
```

Comment:
Oh, I should be giving more details part. Here you don't have to solve the problem, instead just check the value.

So, steps:  

- [ ] check if the /tmp/ip_dgplug.txt exits or not (if not found provide a feedback to the user)
- [ ] read the file and match if it is same of the ip_address you calculated below.

Reply:
> * [X]  check if the /tmp/ip_dgplug.txt exits or not (if not found provide a feedback to the user)
> * [X]  read the file and match if it is same of the ip_address you calculated below.

@kushaldas, I made the required changes following the steps above.